### ISSUE TITLE
German Typo

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmith/republic-at-war#649]
```
In the german Patch, there are a lot of typos.
Maybe I'll fix this, if you get me in the team.

tds157@web[dot]de
```

-----
Original issue reported on code.google.com by `tds...@spamavert.com` on 16 Jun 2011 at 12:43

**Comment by GoogleCodeExporter** [comment]
-----
Original comment by `z3r0...@gmail.com` on 16 Jun 2011 at 4:47
* Changed state: **Invalid**

**Comment by GoogleCodeExporter** [comment]
```
the new fix will come eith the german patch for 1.2 
```

-----
Original comment by `affe...@alpenjodel.de` on 21 Jun 2011 at 8:17
